tools/istio-iptables/pkg/cmd/root.go

```diff
@@ -37,6 +37,13 @@ import (
 
 var (
 	envoyUserVar = env.RegisterStringVar(constants.EnvoyUser, "istio-proxy", "Envoy proxy username")
+	// Enable interception of DNS.
+	// Will be moved to mesh config after it's stable.
+	// TODO: this captures everything, if we want to split cluster.local to TLS and
+	// keep using plain UDP for the rest - we'll need to add another rule to allow
+	// istio-proxy to send.
+	dnsVar = env.RegisterBoolVar("DNS_CAPTURE", true, "If set, address of DNS-over-TLS upstream "+
```

Comment:
Defaults to false in release ?

Reply:
Yes - but true in this PR so it gets covered. Otherwise the tests will just pass.